README.md

```diff
@@ -0,0 +1,172 @@
+<a id="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://user-images.githubusercontent.com/108806646/203941212-a33b97da-6d84-4649-b763-4e8191c24805.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+<h3><b>README</b></h3>
+
+</div>
+
+# 📖Fast Lane Reservations System <a id="about-project"></a>
+
+An app to book a race care to test in the track.
+[Link to the backend](https://github.com/Khaled-AlKhateeb/fast-lane-reservation_backend)
+
+## 🛠 Built With <a id="built-with"></a>
+
+### Tech Stack <a id="tech-stack"></a>
+
+> To build this project we used a tech stack detailed below:
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a id="key-features"></a>
+
+-
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+
+
+## 🚀 Live Demo <a id="live-demo"></a>
+
+- Not Available Yet!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a id="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- To install [Node.js](https://nodejs.org/en/) on your computer.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1. Navigate to a folder where you want the cloned file to appear
+
+2. Clone this repo in your terminal or git bash using the command
+
+   ```
+     git clone <URL> 'https://github.com/Khaled-AlKhateeb/fast-lane-reservations_frontend.git' in your terminal
+      
+   ```
+
+### Install
+
+Install this project with:
+
+run `npm install ` to install all the necessary dependencies including the linters
+
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+### `npm start`
+- This will run the react app in your browser
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### Run tests
+
+To run tests, run the following command:
+
+### `npm run test`
+--->
```

Comment:
- [ ] Instructions about how to run tests have been included in the readme, however, I couldn't find any.  Kindly, either add some tests, or delete this part of the instructions in order to prevent confusion among your users. 👍